package.json

```diff
@@ -61,6 +63,7 @@
     "require-dir": "^0.1.0",
     "run-sequence": "^1.0.2",
     "through2": "^2.0.0",
+    "vinyl": "^0.5.1",
```

Comment:
@cmichaelgraham This should be listed in gulp-typedoc-extractor's dependencies instead of here